build.gradle.kts

```diff
@@ -7,7 +7,7 @@ buildscript {
     dependencies {
         classpath("com.android.tools.build:gradle:7.4.2")
 
-        val kotlinVersion = "1.7.0"
+        val kotlinVersion = "1.9.24"
```

Comment:
Maybe this is the cause

Reply:
@Bohdan-Kim The build is failing because you have JDK 17 installed on your local machine, and the build system expects JDK 11. There are two different ways to resolve this:
* Uninstall JDK 17 from your local machine, and reinstall JDK 11.  Local builds will then work.
* Or, switch the GitHub workflows to build with JDK 17, which the `kotlin-kover-action` is preventing you from doing.  I see that you have a [fork](https://github.com/Bohdan-Kim/kotlin-kover-action/tree/java-version-update) of that action that ups the JDK requirement.  We may need to figure out how to make the GitHub workflows use your fork instead, or else swap out that action for something else that runs Kover.